### DISCUSSION TITLE
VSCode: Cannot find module '@tauri-apps/api/tauri' or its corresponding type declarations

### DISCUSSION THREAD

**Discussion by tbrockman** [tauri-apps/tauri#8348] [Q&A]
**Question:** 
I just created a new Tauri app with `yarn` as my package manager, enabling `Typescript` support, and now VSCode is showing various import errors like `Cannot find module '@tauri-apps/api/tauri' or its corresponding type declarations`. 

How can I fix this?

**Answer:**
If you're not using Yarn classic, you may need to just run the following in the repository root (as was the case for me):

```bash
yarn dlx @yarnpkg/sdks vscode
```

See https://yarnpkg.com/getting-started/editor-sdks for more details.

Didn't find anything specific when I was googling this, so adding this post in case it ends up helping someone else who runs into the same problem.

**Comment by larxn** [comment]
In my case, it was just a matter of installing `@tauri-apps/api`.

**Comment by watter08** [comment]
"@tauri-apps/api": "^2.0.0-rc.0",
import tauriApi from '@tauri-apps/api';
const { invoke } = tauriApi.core;

tauri no longer exists in this version, in my case I needed the invoke and now it is in core

**Comment by NEMYSESx** [comment]
import { invoke } from '@tauri-apps/api/core';
